### ISSUE TITLE
Feature request: allow multiple entity select via Ctrl/Shift-click action

### ISSUE THREAD

**Issue by leungi** [rstudio/sortable#54]
Thanks for this amazingly fun (and useful) 📦!

We used this within a Shiny app recently, and the response was very positive, before the data (to pick from) got longer; at this point, users find it labourious to drag tiles one-by-one, and requested ability to select multiple items at once.

Looking at sortable JS docs, there is a [multi-drag plugin](https://sortablejs.github.io/Sortable/#multi-drag); hope it'll be implemented in R wrapper in the near future 🙏

**Comment by andrie** [comment]
This is already possible, and all that's needed is to include

```r
options = sortable_options(
    multiDrag = TRUE
  )
```

In addition, the CSS should be modified to make it clear that the item is selected.

```r
rank_list(
  text = "You can drag, drap and re-order these items:",
  labels = c("one", "two", "three", "four", "five"),
  input_id = "example_2",
  options = sortable_options(
    multiDrag = TRUE
  )
)
```

I'm adding an example and modified CSS.

**Comment by leungi** [comment]
Thanks for prompt action @andrie 👍 

I pulled from latest commit, but no luck. I tried `Shift`, `Ctrl`, and multi-clicks.

#### Package version
` sortable    * 0.4.2.9000 2020-06-17 [1] Github (rstudio/sortable@26acd36)`

![sortable-multidrag](https://user-images.githubusercontent.com/30273868/84932828-7f92ca00-b09a-11ea-80b9-32281800151f.gif)

**Comment by andrie** [comment]
The problem is that your version doesn't have the modified CSS, so it appears as if nothing is happening.  

None of this is in the master branch yet.  To pick up the work in progress, you'll need to follow along in the linked PR, and perhaps check out commit number 5d162b0823573acb63fa522146a4fdc273fd243d

**Comment by andrie** [comment]
Closed with #55

**Comment by leungi** [comment]
Thanks for clarification @andrie ; should've checked the commit log first and not assume it's pushed to `master`. 🙏
